packages/compass-aggregations/src/modules/aggregation.ts

```diff
@@ -224,4 +225,32 @@ const fetchAggregationData = (page: number): ThunkAction<
   }
 };
 
-export default reducer;
+export const exportAggregationResults = (): ThunkAction<
+  void,
+  RootState,
+  void,
+  Actions
+> => {
+  return (dispatch, getState) => {
+    const {
+      pipeline,
+      namespace,
+      maxTimeMS,
+      collation,
+    } = getState();
+    
+    const stages = pipeline
+      .map(generateStage)
+      .filter((stage) => Object.keys(stage).length > 0);
+    const options: AggregateOptions = {
+      maxTimeMS: maxTimeMS || DEFAULT_MAX_TIME_MS,
```

Comment:
Should we allow users enter a value of `0` for `maxTimeMS` and use it? I'm thinking if users want the[ unbounded default behavior](https://www.mongodb.com/docs/manual/reference/command/aggregate/) they might put in a value of `0` which would be overridden here.